packages/filebrowser/src/listing.ts

```diff
@@ -2237,12 +2246,35 @@ export namespace DirListing {
         node.removeAttribute('data-is-dot');
       }
       // If an item is being edited currently, its text node is unavailable.
+      const indices = !model.indices ? [] : model.indices;
+      let highlightedName = StringExt.highlight(model.name, indices, h.mark);
       if (text) {
-        const indices = !model.indices ? [] : model.indices;
-        let highlightedName = StringExt.highlight(model.name, indices, h.mark);
         VirtualDOM.render(h.span(highlightedName), text);
       }
 
+      // Adds an aria-label to the checkbox element.
+      const checkbox = checkboxWrapper?.querySelector(
+        'input[type="checkbox"]'
+      ) as HTMLInputElement;
+
+      if (checkbox) {
+        checkbox.setAttribute(
+          'aria-label',
+          fileType.contentType === 'directory'
+            ? trans.__(
+                '%1 directory "%2"',
+                selected ? 'Unselect' : 'Select',
+                highlightedName
+              )
+            : trans.__(
+                '%1 file "%2"',
+                selected ? 'Unselect' : 'Select',
+                highlightedName
+              )
```

Comment:
Again, 'unselect' and select are not translated (they would be substituted directly). "Deselect" appears more common form by the way.